Model/Credit_Card_Payment_Method.php

```diff
@@ -121,6 +121,45 @@ public function getConfigFields()
         return array_merge($parentConfigFields, $additionalFields);
     }
 
+    /**
+     * @inheritdoc
+     *
+     * @return Transaction
+     */
+    public function getCancelTransaction(): Transaction
```

Comment:
all of the methods just return the same thing and then all of the paypal getxxxTransaction return new PayPalTransaction()
its not uniform and as it is know it can only be one method.

Reply:
will all post transactions for each payment method, only have one transaction or will there be like a cancel transaction for let's say SEPA need more information in the transaction?
That's why we separated them.

Reply:
Thanks, we changed it now that all the credit card transactions also return a new empty CreditCardTransaction() like in PayPal.